lib/feeder_importer.rb

```diff
@@ -71,25 +160,138 @@ def create_series(pcast = podcast)
     end
 
     self.distribution = Distributions::PodcastDistribution.create!(
-      url: feeder_podcast_url,
+      url: podcast_id,
       distributable: series,
       audio_version_template: template
     )
 
     series
   end
 
-  def retrieve_podcast(a_id = account_id, p_url = feeder_podcast_url)
-    self.podcast = api(root: feeder_root, account: a_id).tap { |a| a.href = p_url }.get
+  def create_series_image(podcast_image, purpose)
+    upload_url = copy_image(SecureRandom.uuid, podcast_image)
+    image = series.images.create!(upload: upload_url, purpose: purpose)
+    announce_image(image)
+    podcast_image.update_attribute(:original_url, image.public_url(version: 'original'))
+  end
+
+  def create_stories
+    podcast.episodes.each do |episode|
+      create_story(episode)
+    end
+  end
+
+  def create_story(episode)
+    # puts "episode.categories: #{episode.categories.inspect}"
+    attrs = {
+      app_version: PRX::APP_VERSION,
+      creator_id: user_id,
+      account_id: account_id,
+      title: episode.title,
+      short_description: episode.subtitle,
+      description_html: episode.description,
+      tags: episode.categories,
+      published_at: episode.published_at
+    }
+    story = series.stories.create!(attrs)
+
+    version = story.audio_versions.create!(
```

Comment:
in regular import we apply a label here, right?

Reply:
sure!